projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -1964,6 +1999,20 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "// TODO:Spartacus - Property 'userReplenishmentOrderService' was removed from Class 'ReplenishmentOrderHistoryComponent'. ",
     },
+    {
+      class: 'TrackingEventsComponent',
+      importPath: '@spartacus/order/components',
+      deprecatedNode: 'activeModal',
```

Comment:
this is constructor change, and it is already in `5_0-generated-typescirpt-changes-doc.md`

Reply:
A constructor argument also declares a corresponding property.  Hence it is expected to have migrations for both the updated constructor signature and for the removed property.